schemas/control-statement-schema.json

```diff
@@ -84,7 +84,7 @@
     "retrieved_at": {
       "description": "Date-time this was retrieved from the source",
```

Comment:
This description also needs updating. Maybe just rephrase to remove any mention of the type, so it's not coupled.